cookiecutter.json

```diff
@@ -8,19 +8,14 @@
     "top_level_domain_name": "com",
     "second_level_domain_name": "crowdbotics",
     "project_package_name": "{{ cookiecutter.project_slug|lower|replace(' ', '')|replace('-', '') }}",
-    "entity_id_type": "String",
-    "has_h2_database": "y",
+    "database_option": ["h2", "mysql", "postgresql", "mongodb", "none"],
+    "entity_id_type": ["Long", "String"],
     "has_lombok": "n",
-    "has_mongodb_database": "n",
-    "has_mysql_database": "n",
-    "has_postgresql_database": "n",
-    "has_redis_database": "n",
     "has_spring_boot_devtools": "y",
     "has_spring_boot_jms": "n",
-    "has_spring_boot_ldap": "n",
-    "has_spring_boot_oauth2": "y",
     "has_spring_boot_rest": "n",
     "has_spring_boot_session": "n",
     "java_version": "1.8",
+    "spring_boot_authentication": ["basic", "jwt", "ldap", "oauth2"],
```

Comment:
I don't know much about Spring but couldn't you have more than one of these? For options that can have multiple values they need to remain separate.

Reply:
I guess you could, but I don't see the point.

Reply:
I suspect we'll have to break these out into separate options again in the future but in the interest of getting this merged it's fine for now. I can think of use cases for multiple methods, for example JWT and/or OAuth2 for the API and basic for the website.